src/models/domain_period_discover_api_host.rs

```diff
@@ -449,7 +455,7 @@ pub struct DomainPeriodDiscoverApiHost {
     /// A list of network ids to which host belongs
     #[serde(rename = "ot_network_ids", skip_serializing_if = "Option::is_none")]
     pub ot_network_ids: Option<Vec<String>>,
-    /// A list of serial numbers that discovered with host
+    /// A list of ot serial numbers that discovered with host
```

Comment:
This update reintroduces this, is this a spelling mistake or it should be `ot`?

Reply:
I believe that this should be `ot` -> https://www.redhat.com/en/topics/edge-computing/what-is-ot